src/settings.c

```diff
@@ -887,7 +887,7 @@ static int setting_perform_request_reply_from(settings_t *ctx,
   registration_state_deinit(&ctx->registration_state);
 
   if (!success) {
-    ctx->api_impl.log(log_err, "setting req/reply failed for msg id %d", message_type);
+    ctx->api_impl.log(log_err, "setting req/reply failed after %d retries (msg id: %d)", tries, message_type);
```

Comment:
This is ok as error?

Reply:
Good catch, fixed to warning as intended